### PR TITLE
David product

### PR DESCRIPTION
**Pull request by davidnicholastaylor**
[Link](https://github.com/NSS-Likeable-Lemurs/BangazonAPI/issues/4)

- Created model for Product 
- Created controller for Product

```
git fetch --all
git checkout davidProduct
```
Test POST, PUT, DELETE, and GET in POSTMAN

Created model and controller will affect both customer and product type models and controllers.

@Akeen11 
@gretchenmarie 
@KaylaReid 
@askingalot

### CODE REVIEW

**Review by Akeen11 — APPROVED** [review]
👍

File: Bangazon/Controllers/ProductsController.cs [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+
+        // GET api/students?q=Taco
+        [HttpGet]
+        public async Task<IActionResult> Get(string q)
```
Comment (askingalot) [inline Bangazon/Controllers/ProductsController.cs]: There is no requirement to be able to search Products with a `q` parameter.

File: Bangazon/Controllers/ProductsController.cs [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+                    OR p.Title LIKE '%{q}%'
+                    OR p.Description LIKE '%{q}%'
+                    OR p.Quantity LIKE '%{q}%'
+                    OR p.ProductTypeId LIKE '%{q}%'
+                    OR p.CustomerId LIKE '%{q}%'
+                ";
+                sql = $"{sql} {isQ}";
+            }
+
+            Console.WriteLine(sql);
```
Comment (askingalot) [inline Bangazon/Controllers/ProductsController.cs]: Remove `Console.WriteLine()`

**Review by askingalot — CHANGES_REQUESTED** [review]
Just a couple of small things...

**Review by askingalot — APPROVED** [review]

**Pull request merged**